clusterloader2/testing/access-tokens/config.yaml

```diff
@@ -42,7 +42,11 @@
 {{$RESTART_COUNT_THRESHOLD_OVERRIDES:= DefaultParam .RESTART_COUNT_THRESHOLD_OVERRIDES ""}}
 
 name: access-tokens
-automanagedNamespaces: {{$namespaces}}
+namespace:
+  count: {{$namespaces}}
+  prefix:
+  deleteStaleNamespaces: false
+  deleteAutomanagedNamespaces: true
```

Comment:
I really don't like the fact we need to set those values in every test - we need to smartly default them instead and rely on the defaulting whenever it is possible.

Reply:
OK, will set the following default values. 
```
count=1
prefix= ""
deleteStaleNamespaces=false
deleteAutomanagedNamespace=true
```

Reply:
removed